test/integration/kubernetes_deploy_test.rb

```diff
@@ -21,7 +21,8 @@ def test_full_hello_cloud_set_deploy_succeeds
       %r{Service/web\s+Selects at least 1 pod},
       %r{DaemonSet/ds-app\s+1 updatedNumberScheduled, 1 desiredNumberScheduled, 1 numberReady},
       %r{StatefulSet/stateful-busybox},
-      %r{Service/redis-external\s+Doesn't require any endpoint}
+      %r{Service/redis-external\s+Doesn't require any endpoint},
+      %r{Job/hello-job\s+Exists}
```

Comment:
Can we be more helpful about this? Our success condition is more complex than this implies. For example, we might say "Succeeded" or "Started" as applicable.